metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/api/SraObjectsImporterThroughApi.java

```diff
@@ -108,6 +108,7 @@ protected Study extractAnalysisFromStudy(StudyType studyType, Study study) throw
             } catch (Exception exception) {
                 IMPORT_LOGGER.log(Level.SEVERE, "Encountered Exception for accession " + analysisAccession);
                 IMPORT_LOGGER.log(Level.SEVERE, exception.getMessage());
+                exception.getCause().printStackTrace();
```

Comment:
I recommend to print the stacktrace through the logger, to make sure it is close to the 2 previous messages.

Reply:
I just didn't know how to do this using the logger. Turns out I can just do

```
IMPORT_LOGGER.log(Level.SEVERE, exception.getMessage(), exception);
```
And it will log both the message and the stack trace automatically.

Reply:
Addressed in https://github.com/EBIvariation/metadata-ws/pull/86/commits/608ed7c7bf51ffab28ecd9637833f8cf5bf94477.